nodejs unconditionally links to libatomic

Clang implicitly links to atomic and on systems without libgcc this
library is not installed, which breaks the build.

See: https://github.com/nodejs/node/commit/4bd0d8c22ca65ba2bd02c671048a99e414f049b8
See: https://bugs.gentoo.org/869992
---
 node.gyp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/node.gyp b/node.gyp
index e8227bb2..4bc53925 100644
--- a/node.gyp
+++ b/node.gyp
@@ -117,9 +117,6 @@
           '-Wl,-bnoerrmsg',
         ],
       }],
-      ['OS == "linux" and llvm_version != "0.0"', {
-        'libraries': ['-latomic'],
-      }],
     ],
   },
 
-- 
2.37.3

